src/main/scala/spray/boilerplate/BoilerplatePlugin.scala

```diff
@@ -65,7 +65,8 @@ object BoilerplatePlugin extends AutoPlugin {
     clearTargetDir(streams, targetDir, signature, newFiles)
     mapping foreach {
       case (templateFile, target) =>
-        if (templateFile.lastModified > target.lastModified) {
+        // the target.lastModified == 0 relates to https://github.com/sbt/sbt-boilerplate/issues/86
+        if (templateFile.lastModified > target.lastModified || target.lastModified == 0) {
```

Comment:
the problem in #86 was that `lastModified` is 0 for non-existing files, causing trouble when the template file has had its `lastModified` cleared.

Might be clearer to do:


```suggestion
        if (!target.exists || templateFile.lastModified > target.lastModified) {
```

?